.fossa.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019-present Open Networking Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
SPDX-License-Identifier: Apache-2.0 ?? should we add line in this file as well ?

Reply:
done